README.md

```diff
@@ -28,15 +28,18 @@ All new backend code and React components should be well tested.
 
 ## Supported operating systems
 
-* Ubuntu 18.04, 20.04, 22.04 or equivalent
+* Ubuntu 18.04, 20.04 or equivalent
+* [Manjaro](manjaro.md)
 
 ## Prerequisites
 
 * Node 14
 * Yarn
 * PostgreSQL 10  or 12
 * Ruby 2.7
-* Ubuntu 18.04, 20.04, 22.04 or equivalent
+
+* Ubuntu 18.04, 20.04 or equivalent
+* Manjaro
```

Comment:
Let's remove this. I think it's clear that if you choose Manjaro, you're making the decision to figure this out on your own.